### PR TITLE
[KOA-5091] Overlay SwiftUI

### PR DESCRIPTION
**Pull request by frugoman**
Introducing the BPKOverlay View modifier

image

### CODE REVIEW

File: Backpack-SwiftUI/Overlay/Classes/BPKOverlay.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
+
```
Comment (gert-janvercauteren) [inline Backpack-SwiftUI/Overlay/Classes/BPKOverlay.swift:1]: Can you add the header comment?

File: screenshots/iPhone-swiftui_overlay___bottom_lm.png
Comment (gert-janvercauteren) [inline screenshots/iPhone-swiftui_overlay___bottom_lm.png:1]: Should these docs screenshots be 'included' in the UIKit example? There's no tab-bar currently in view?

**Review by gert-janvercauteren — COMMENTED** [review]
Code looks fine.
We should probably align the docs screenshots. (UIKit <> SwiftUI)

File: screenshots/iPhone-swiftui_overlay___bottom_lm.png
Comment (frugoman) [reply screenshots/iPhone-swiftui_overlay___bottom_lm.png:1]: the tab bar shows on the List, not in the actual example view! (same as in the uikit one)

File: Backpack-SwiftUI/Overlay/Classes/BPKOverlay.swift (hunk 1 above)
Comment (frugoman) [reply Backpack-SwiftUI/Overlay/Classes/BPKOverlay.swift:18]: Added!

**Review by gert-janvercauteren — APPROVED** [review]

**Pull request merged**